THIRD_PARTY_NOTICES.md

```diff
@@ -5,6 +5,19 @@ Without those this repository would never been created. ❤ Much love.
 
 ---
 
+<details>
+<summary>Json.NET Converters &copy; 2018 Wanzyee Studio</summary>
+
+```
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[fenced-code-flag] Missing code language flag](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5461512)